src/sagemaker/estimator.py

```diff
@@ -1816,6 +1838,11 @@ def __init__(
                 will be disabled (default: ``False``).
             environment (dict[str, str]) : Environment variables to be set for
                 use during training job (default: ``None``)
+            max_retry_attempts (int): The number of times to move a job to the STARTING status.
+                You can specify between 1 and 30 attempts.
+                If the value of attempts is greater than one, the job is retried on InternalServerFailure the same number of attempts as the value.
+                You can cap the total duration for your job by setting ``max_wait`` and ``max_run``
+                (default: ``None``)
```

Comment:
Please explain here that if this is set to None there will be no retries.

Reply:
i am wondering if we should set the default to 3 or 2.

Reply:
We can set the default to 3 or 2 as we want to customers to explicitly opt-in. However I think I need to re-word it like " If the value of attempts is greater than zero, the job is retried on InternalServerFailure the same number of attempts as the value.
                 You can cap the total duration for your job by setting ``max_wait`` and ``max_run``"

Reply:
yeah, that is more clear. Let's just do None for the default for now.

Reply:
done